robottelo/hosts.py

```diff
@@ -1102,6 +1102,24 @@ def install_tracer(self):
             raise ContentHostError('There was an error installing katello-host-tools-tracer')
         self.execute('katello-tracer-upload')
 
+    def register_to_cdn(self):
+        """Subscribe satellite to CDN"""
+        self.remove_katello_ca()
+        major_version = self.os_version.major
+        release_version = f'{major_version}Server' if major_version < 8 else f'{major_version}'
+        cmd_result = self.register_contenthost(
+            org=None,
+            lce=None,
+            username=settings.subscription.rhn_username,
+            password=settings.subscription.rhn_password,
+            releasever=release_version,
+        )
+        if cmd_result.status != 0:
+            raise ContentHostError(
+                f'Error during registration, command output: {cmd_result.stdout}'
```

Comment:
Even this should go into `register_contenthost`.

Reply:
I'm not sure if this should be added to register_contenthost, as register_contenthost simply returns `return self.execute(cmd)`, and including this check there is a good idea, I think this will make this PR more huge, so I prefer this to handle in separate PR